targets/lenovo-x1/netvmExtraModules.nix

```diff
@@ -18,6 +18,98 @@
     );
   };
 
+  netvmAdditionalFirewallConfig = let
+    externalNic = "wlp0s5f0";
```

Comment:
Please inherit and reuse this https://github.com/tiiuae/ghaf/blob/eefaa2dd2865a36760d5ca07453bde7d1e2496b8/modules/common/hardware/lenovo-x1/definitions/x1-gen11.nix#L29 instead of redefining it here.